apps/client/src/common/components/multi-part-progress-bar/MultiPartProgressBar.tsx

```diff
@@ -19,8 +19,8 @@ export default function MultiPartProgressBar(props: MultiPartProgressBar) {
 
   const percentComplete = 100 - clamp(100 - (Math.max(now ?? 0, 0) * 100) / complete, 0, 100);
 
-  const dangerWidth = clamp((danger / complete) * 100, 0, 100);
-  const warningWidth = clamp((warning / complete) * 100, 0, 100);
+  const dangerWidth = clamp(((danger || 0) / complete) * 100, 0, 100);
+  const warningWidth = clamp(((warning || 0) / complete) * 100, 0, 100);
```

Comment:
would this not be 0 if there is no warning / danger? could we instead write

```ts
const dangerWidth = danger ? clamp(((danger) / complete) * 100, 0, 100) : 0
```

Reply:
yours is easier to read so sure i'll go with that